pkg/bosh/bpmconverter/volume_factory.go

```diff
@@ -282,6 +328,42 @@ func (f *VolumeFactoryImpl) GenerateBPMDisks(manifestName string, instanceGroup
 	return bpmDisks, nil
 }
 
+func generateEphemeralPersistentVolumeClaim(manifestName string, instanceGroup *bdm.InstanceGroup, namespace string) corev1.PersistentVolumeClaim {
+	// Spec of a persistentVolumeClaim
+
+	diskSize := defaultEphemeralVolumeSize
```

Comment:
Just nitpicking: generating the disk size could be a separate private function. I don't mind keeping it inline, but it looks like it really belongs to the instance group struct and we tend to amass these procedural, intervined if blocks in our 'generate' funcs :)